acm/src/main/java/org/literacybridge/acm/gui/ResourceView/audioItems/AudioItemTableModel.java

```diff
@@ -11,22 +11,14 @@
 import org.literacybridge.acm.gui.util.UIUtils;
 import org.literacybridge.acm.gui.util.language.LanguageUtil;
 import org.literacybridge.acm.repository.AudioItemRepository.AudioFormat;
-import org.literacybridge.acm.store.AudioItem;
-import org.literacybridge.acm.store.Committable;
-import org.literacybridge.acm.store.MetadataSpecification;
-import org.literacybridge.acm.store.MetadataStore;
+import org.literacybridge.acm.store.*;
```

Comment:
I've read suggestions that it is better to at least prefer explicit imports over wild-card imports, to prevent accidentally importing an unintended class or name. Given how libraries contain classes with the same name (like Date), I tend to agree.

It doesn't matter so much with our internal classes, but I've got my IDE set up to do the expansion, so I get it for all imports.

Reply:
Yeah, that was my IDE. I changed my settings.